built_value_generator/lib/built_value_generator.dart

```diff
@@ -71,6 +71,7 @@ class BuiltValueGenerator extends Generator {
           '// ignore_for_file: unnecessary_const\n'
           '// ignore_for_file: unnecessary_new\n'
           '// ignore_for_file: test_types_in_equals\n'
+          '// ignore_for_file: avoid_as\n'        
```

Comment:
This is starting to get quite long.

`ignore_for_file` allows multiple values separated by a comma; I suggest packing them all onto one line, in alphabetical order:

```
`// ignore_for_file: always_put_control_body_on_new_line,'
'annotate_overrides,'
'avoid_annotating_with_dynamic,'
...
```

What do you think? Thanks.

Reply:
I don't think is a good idea to store all in one line. Even it's a generated file, it will make hard to 
 users to read it. 
What are your thoughts?

Reply:
With that in mind I will put the ignore statements in the bottom of the file.